server/services/offenderService.test.ts

```diff
@@ -83,6 +83,80 @@ describe('getOffenders', () => {
       expect(elite2Client.getLocations).toBeCalledWith('WRI')
     })
 
+    it('should sort retrieved locations with top 2 primary locations at the begining', async () => {
+      elite2Client.getLocations.mockReturnValue([
+        { id: 2, userDescription: 'place 2' },
+        { id: 'o', userDescription: 'Other cell' },
```

Comment:
the ids that are returned from elite are numeric, doesn't affect your tests too much but would be good to keep them consistent with the real types